src/type.ts

```diff
@@ -18,6 +18,9 @@ function typeORMColumnTypeToGraphQLType(columnType: TypeORM.ColumnType) {
     columnType === 'varying character' ||
     columnType === 'char' ||
     columnType === 'character' ||
+    columnType === 'longtext' ||
```

Comment:
Could you add `longtext`, `mediumtext`, `tinyblob`, `mediumblob`, `blob`, `nchar`, `national char`, `nvarchar`, `national varchar` here too?